pkg/volume/glusterfs/glusterfs.go

```diff
@@ -145,6 +157,29 @@ func (plugin *glusterfsPlugin) execCommand(command string, args []string) ([]byt
 	return cmd.CombinedOutput()
 }
 
+func (plugin *glusterfsPlugin) readClusterConf() (*glusterfsClusterConf, error) {
```

Comment:
you should parse `glusterfsClusterConf` out of `StorageClass.Parameters`, not from a file.